app.js

```diff
@@ -102,7 +104,7 @@ const submitBtn = function submit() {
 
     pickedTime.val();
 
-    //
+    // check to make sure all requirements are met so they can go to the next page
    if( (userDate == "" ) || (userState == "") || (pickedTime == "") || (userCity == "")){
```

Comment:
Change this to individual conditions, with `status = true;` by default, and if any condition is true, it sets `status = false;`